### PR TITLE
Fixing AgentToolsPath variable syntax

### PR DESCRIPTION
**Pull request by jcagme**
Include the begin and end extension tasks.

skip ci please

### CODE REVIEW

File: buildpipeline/Core-Setup-Windows-BT.json [hunk 1]
```diff
@@ -1,5 +1,28 @@
...
+        "arguments": "",
+        "script": "if (Test-Path \"$Env:AgentToolsPath\\begin.ps1\") {\n   \"$Env:AgentToolsPath\\begin.ps1 script found. Executing...\"\n    & $Env:AgentToolsPath\\begin.ps1\n} else {\n   \"$Env:AgentToolsPath\\begin.ps1 script does not exist. Moving on...\"\n}",
+        "errorActionPreference": "continue",
+        "failOnStderr": "false",
+        "ignoreLASTEXITCODE": "true",
+        "workingDirectory": ""
+      }
+    },
     {
       "enabled": true,
```
Comment (weshaggard) [inline buildpipeline/Core-Setup-Windows-BT.json]: What about the removal of the VSTS agent clean-up step?

Comment (jcagme) [reply buildpipeline/Core-Setup-Windows-BT.json]: I'm working on https://github.com/dotnet/core-eng/issues/3320 but can I include this work in here too. On it...

**Comment by weshaggard** [comment]
@zamont is OK to merge infra changes into uwp release branch?

**Comment by zamont** [comment]
Yes feel free to merge infra changes

**Comment by jcagme** [comment]
@zamont should we also do UWP6.1?

**Comment by zamont** [comment]
uwp6.1 release is in the uwp-setup repo. Actually, we have a uwp6.0 branch there so you could instead make these changes to that uwp6.0 branch instead of this core-setup branch

**Comment by jcagme** [comment]
Do we run official builds off of UWP-Setup too?  Do we do official build from both?

**Comment by zamont** [comment]
uwp-setup doesn't ship with the rest of .NET Core, so I believe the answer is no. But that is our shipping vehicle for UWP.

**Comment by jcagme** [comment]
Will do this work for UWP in a different branch (repo?) once I figure out what makes sense since I dis not find a UWP core-setup build since November of 2017

**Pull request closed without merging**